src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md

```diff
@@ -54,12 +56,19 @@ The following examples enable the remote storage with an AWS S3 storage adapter
    bin/magento setup:config:set --remote-storage-driver="aws-s3" --remote-storage-bucket="myBucket" --remote-storage-region="us-east-1"
    ```
 
+-  Install new Magento with S3 compatible remote storage
+
+   ```bash
+   bin/magento setup:install --remote-storage-driver="aws-s3" -- remote-storage-endpoint="
+   ttp://localhost:9000" --remote-storage-bucket="myBucket" --remote-storage-region="us-east-1" --remote-storage-path-style=1
```

Comment:
Shouldn't this be "http://..." or am I missing something?

Reply:
Absolutely